integrations/tensorflow/bindings/python/pyiree/tf/support/tf_test_utils_test.py

```diff
@@ -78,6 +101,60 @@ def test_recursive_check_same(self, array_c, array_d, array_e, tgt_same):
     same = tf_test_utils._recursive_check_same(ref, tgt)
     self.assertEqual(tgt_same, same)
 
+  def test_trace_inputs_and_outputs(self):
+    backend_info = tf_utils.BackendInfo.ALL['tf']
+    module = backend_info.CompiledModule(StatefulCountingModule, backend_info)
+
+    def trace_function(trace):
```

Comment:
rather than calling this argument "trace", it should be just "module" (or "m" or "mod"). That is, the function effectively just does some computation on a module. If the module happens to be a TracingModule, then the calling code can obtain a Trace as well.

Reply:
SGTM.